docs/eigenlayer/operator-guides/key-management/institutional-operators.md

```diff
@@ -6,5 +6,10 @@ sidebar_position: 2
 
 - Secure keys, including secrets such as passphrases or mnemonics, using services like AWS Secrets Manager or Hashicorp Vault. These services can be seamlessly integrated with automated mechanisms that safely retrieve secrets or keys (e.g., remote signers). If resources permit, consider running your own Hashicorp Vault instance, which grants full custody of keys and secrets while sacrificing the service provider's availability and security guarantees.
 - Avoid generating all keys with the same mnemonic. Minimize the attack surface by employing a new mnemonic for every 200 or 1000 validator keys, depending on your preference. This approach also reduces the risk of losing key generation capabilities if a single mnemonic is lost, and limits the impact if an attacker gains access to a few mnemonics.
+- Given that Avs keys are likely to be much fewer, not using a seed to generate the keys is probably safer; generate each avs key independently if possible.
```

Comment:
`not using a seed to generate the keys is probably safer` still not get why?

Reply:
Can we capitalize all uses of "AVS" for consistency?

Reply:
@wesfloyd [d2e4ea5](https://github.com/Layr-Labs/eigenlayer-docs/pull/218/commits/d2e4ea5a0fe82470fb1673b0292a870374dbee78)

Reply:
@shrimalmadhur not sure if we should include that. Thinking is that if all your keys are generated from the same seed and you lose that seed they get all your keys. But if the keys are unrelated and they manage to steal one at least the others are safe.

Reply:
`not using a same seed to` ? add `same` ?

Reply:
[de4d6cb](https://github.com/Layr-Labs/eigenlayer-docs/pull/218/commits/de4d6cbf68803c22cfc05e26c323a6b9dd3f408f)